controller/src/beerocks/bml/rdkb/bml_rdkb_defs.h

```diff
@@ -92,15 +92,15 @@ struct BML_STEERING_RRM_CAPS {
 };
 
 typedef enum {
```

Comment:
I know there is no place currently using the existing `DISCONNECT_SOURCE_*` and `DISCONNECT_TYPE_*` literally - and hence the bml_rdkb_defs.h change is enough - but I felt the need to verify that in the code myself.
Perhaps the commit message should specify that to clarify no calling locations changes are required.
(nitpick)